Dockerfile.tests-extension

```
@@ -0,0 +1,34 @@
+    rm -f oc-mirror-tests-ext
+
+# Final image containing the test extension binary
+FROM registry.ci.openshift.org/ocp/4.17:base-rhel9
+
+# Copy the compressed test extension binary
+COPY --from=test-extension-builder \
+    /go/src/github.com/openshift/oc-mirror/tests-extension/bin/oc-mirror-test-extension.tar.gz \
+    /usr/bin/
+
+LABEL io.openshift.release.operator=true \
+      com.redhat.component="
```

Hadolint (2.14.0)

[error] unexpected newline
expecting '"' or '\'

(DL1000)